### DISCUSSION TITLE
Data Viz for sqlite on MacOs

### DISCUSSION THREAD

**Discussion by Rchop49** [sqlitebrowser/sqlitebrowser#3380] [Q&A]
Is there a common tool (free or subscription needed) recommended for data visualization that works well with SQLite on a Mac? I'm fairly new to SQL and coding in general but I've heard of things like Tableau and PowerBI. Just curious if there is a common suggestion to start playing around with. Thanks!

**Comment by scottfurry** [comment]
I get a little ticked not getting an answer back on a question. So, please accept apologies that this issue was not answered more quickly.

Bit of a sad commentary that a web search of "sqlite data visualization" results in the first link returned being this project. _Hmm..._

If you're looking for help on sqlite, I would encourage you to have a look at the [SQLite Tutorial site](https://www.tutorialspoint.com/sqlite/index.htm). That should get you going in a good direction. Our wiki, and other docs, need a bit of improvement. Suggestions welcomed.

**Comment by justinclift** [comment]
Oh, I somehow missed this question as well.  My personal recommendation would be [Redash](https://github.com/getredash/redash), though that'd be in a few weeks.  The project is being relaunched as a Community Led effort, and we're not quite finished getting things cleaned up and ready for production use.  It'll be fine in a few weeks though. :smile:
